cmd/tempo/app/app.go

```diff
@@ -163,6 +173,35 @@ func (c *Config) CheckConfig() {
 	}
 }
 
+func (c *Config) Describe(ch chan<- *prometheus.Desc) {
+	ch <- metricConfigFeatDesc
+}
+
+func (c *Config) Collect(ch chan<- prometheus.Metric) {
+
+	features := map[string]int{
+		"external_endpoints": 0,
```

Comment:
I think `search_external_endpoints` would be more clear, WDYT?

Reply:
Good call, that sounds good to me.